pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -76,7 +80,7 @@ for arg; do
     else
         case $argfun in
             set_*)
-                var=$(echo $argfun | sed 's,^set_,,')
+                var=${argfun/#set_/}
```

Comment:
I tested in zsh, bash, and sh version of bash, and none seems to work as expected.